test/box/gh-6405-discarding-arguments-in-box-func-call.result

```diff
@@ -0,0 +1,190 @@
+-- test-run result file version 2
+env = require('test_run')
+ | ---
+ | ...
+test_run = env.new()
+ | ---
+ | ...
+
+test_run:cmd("create server test with script = \"box/lua/cfg_test1.lua\"")
```

Comment:
Why do you need another instance with `cfg_test1.lua`?

Reply:
fixed